src/layer.rs

```diff
@@ -62,7 +65,15 @@ impl Layer {
             })
             //FIXME: come up with a better way of reporting errors than just aborting at first failure
             .collect::<Result<_, _>>()?;
-        Ok(Layer { contents, glyphs })
+
+        let layerinfo_path = path.join(LAYER_INFO_FILE);
+        let info = if layerinfo_path.exists() {
+            LayerInfoData::from_file(&layerinfo_path)?
+        } else {
+            LayerInfoData::default()
```

Comment:
does this mean that if we load a file that has no layer info, and then save, we will save out an empty `layerinfo.plist`?

Reply:
It shouldn't. https://github.com/linebender/norad/pull/67/files/0823056311edf80b71c4b350b6ea4b7cf0ff5f79#diff-5eab58a7f7ef848629a8be1084b12464ced845ebd69087b9cbc1a2263465212aR86-R88